### PR TITLE
BGDIINF_SB-2835: Improved docu in spec about compression

### PR DESCRIPTION
**Pull request by ltshb**
Spec can be tested here : 
- https://sys-data.dev.bgdi.ch/api/stac/static/spec/v0.9/api.html
- https://sys-data.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#tag/Asset-Upload-Management

### CODE REVIEW

**Review by boecklic — APPROVED** [review]

File: spec/transaction/tags.yaml [hunk 1]
```diff
@@ -9,13 +9,108 @@ tags:
...
+
+      POST/PUT requests require authentication as described in [here](#tag/Authentication).
+
+      ## HTTP Compression
+
+      Text based files between *1 MB* and *10 MB* are automatically compressed during download using *gzip* or *br* based on the
+      `Accept-Encoding` header of the request. But note that this compression is only done for standard Media Type (see
+      [File types that CloudFront compresses](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html#compressed-content-cloudfront-file-types)).
+
+      <span style="color: red">It is highly recommended to not upload text based files bigger than *10 MB* !</span>
```
Comment (boecklic) [inline spec/transaction/tags.yaml:93]: I'd probably formulate this a bit different and less 'dramatic': the use case of having big files on stac is in general not uncommon, so we can not say it's highly unrecommended. 
The general recommendation for files bigger than 10Mb, regardless of the type, should be to use a compressed media type. If this is not possible (as e.g. for jsons directly used in a browser app), the file should be compressed upfront or split.

File: spec/transaction/tags.yaml [hunk 2]
```diff
@@ -9,13 +9,108 @@ tags:
...
     description: |
-      Asset file can be uploaded via the STAC API using the following requests.
+      Asset files are uploaded via the STAC API using the API requests described in this chapter.
+
+      - [Authentication](#section/Authentication)
+      - [Compression](#section/Compression)
+      - [Example](#section/Example)
+
+      <span style="color:red">***IMPORTANT NOTES:***</span>
+        - <span style="color:red">*Text based files bigger than 10 MB should use HTTP compression,
```
Comment (boecklic) [inline spec/transaction/tags.yaml:79]: see comment below

**Review by boecklic — APPROVED** [review]

**Review by hansmannj — APPROVED** [review]
:+1: Thanks!

**Comment by ltshb** [comment]
@boecklic I've adapted the spec as discussed you can have a look at it here https://sys-data.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#tag/Asset-Upload-Management

**Comment by boecklic** [comment]
> @boecklic I've adapted the spec as discussed you can have a look at it here https://sys-data.dev.bgdi.ch/api/stac/static/spec/v0.9/apitransactional.html#tag/Asset-Upload-Management

Perfect! Thanks.

**Review by boecklic — APPROVED** [review]

**Pull request merged**